bower.json

```diff
@@ -20,10 +20,10 @@
   "dependencies": {
     "purescript-effect": "^3.0.0",
     "purescript-functions": "^5.0.0",
-    "purescript-maybe": "^5.0.0"
+    "purescript-maybe": "^5.0.0",
+    "purescript-prelude": "^5.0.0"
```

Comment:
Shouldn't prelude be `5.0.1`?

Reply:
What I did was check the .spago directory contents after doing a spago build: 
![image](https://user-images.githubusercontent.com/15835006/127256838-929fec18-80ca-4040-9a81-007759d76552.png)